instrumentor/controllers/instrumentationdevice/common.go

```diff
@@ -64,15 +64,14 @@ func isDataCollectionReady(ctx context.Context, c client.Client) bool {
 	return nodeCollectorsGroup.Status.Ready
 }
 
-func addInstrumentationDeviceToWorkload(ctx context.Context, kubeClient client.Client, instConfig *odigosv1.InstrumentationConfig) (error, bool) {
-	// devicePartiallyApplied is used to indicate that the instrumentation device was partially applied for some of the containers.
-	devicePartiallyApplied := false
-	deviceNotAppliedDueToPresenceOfAnotherAgent := false
+func addInstrumentationDeviceToWorkload(ctx context.Context, kubeClient client.Client, instConfig *odigosv1.InstrumentationConfig) error {
```

Comment:
consider giving this function a better name now that it's no longer addressing instrumentation device